python/metashape_workflow_functions.py

```diff
@@ -1002,7 +1010,8 @@ def build_dem_orthomosaic(self):
                         source_data=Metashape.ElevationData,
                         image_compression=compression,
                     )
-
+                    self.written_paths[f"DEM_{self.cfg['buildDem']['surface'][0]}"] = output_file #export
```

Comment:
nitpick: it could be useful to have a comment explaining the logic of how/why you are constructing the dict key.